packages/client/src/components/MemoryGameCard/MemoryGameCard.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { theme as t, CardProps, Card, Icon } from '@codement/ui';
+
+export interface MemoryGameCardProps extends CardProps {
+    flipped?: boolean;
+    state?: 'correct' | 'incorrect';
+    content?: string;
+}
+
+const StyledCard = styled(Card)<MemoryGameCardProps>`
+  position: relative;
+  width: 160px;
+  height: 160px;
```

Comment:
Can we please use a `${theme.size()}` value here, or at least a rem? 🙏

Reply:
${theme.size('missive')} is 7.2rem which is not big enough. So changed to 16rem.